cmd/seautil/cmd/certs.go

```diff
@@ -33,25 +33,22 @@ type Flag struct {
 	CertEtcdPath string
 }
 
-var config *Flag
-
-// certsCmd represents the certs command
-var certsCmd = &cobra.Command{
-	Use:   "certs",
-	Short: "generate kubernetes certs",
-	Long:  `seautil cert --node-ip 192.168.0.2 --node-name master1 --dns-domain aliyun.com --alt-names aliyun.local`,
-	RunE: func(cmd *cobra.Command, args []string) error {
-		nodeIP := net.ParseIP(config.NodeIP)
-		if nodeIP == nil {
-			return fmt.Errorf("input --node-ip(%s) is an invalid IP format", config.NodeIP)
-		}
-		return cert.GenerateCert(config.CertPath, config.CertEtcdPath, config.AltNames, nodeIP, config.NodeName, config.ServiceCIDR, config.DNSDomain)
-	},
-}
+// NewCertGenCmd gen all kubernetes certs
+func NewCertGenCmd() *cobra.Command {
+	config := new(Flag)
 
-func init() {
-	config = &Flag{}
-	rootCmd.AddCommand(certsCmd)
+	var certsCmd = &cobra.Command{
+		Use:   "gen",
+		Short: "generate kubernetes certs",
+		Long:  `seautil cert gen --node-ip 192.168.0.2 --node-name master1 --dns-domain aliyun.com --alt-names aliyun.local --service-cidr 10.103.97.2/24`,
```

Comment:
No, please remove the aliyun-related details. sealer is a universe tool, it should not add commercial related content in its code.